packages/renderer/src/lib/ui/ProviderInfoCircle.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+let { type }: { type: 'kubernetes' | 'podman' | 'docker' | undefined } = $props();
+let color = $state('');
+
+// Each provider has a colour associated to it within tailwind, this is a map of those colours.
+// bg-purple-600 = podman
+// bg-sky-300    = docker
+// bg-sky-600    = kubernetes
+// bg-gray-900   = unknown
+$effect(() => {
+  if (type === 'podman') {
+    color = 'bg-purple-600';
+  } else if (type === 'docker') {
+    color = 'bg-sky-400';
+  } else if (type === 'kubernetes') {
+    color = 'bg-sky-600';
+  } else {
+    color = 'bg-gray-900';
+  }
```

Comment:
switch ?

Reply:
AFAIK it won't work with a switch on the type

Reply:
(I think I had the switch at some point)

Reply:
updated